hw/ip/adc_ctrl/doc/programmers_guide.md

```diff
@@ -45,13 +45,14 @@ The debug accessory voltage range of interest is shown in the diagram below:
 The ADC can be used to detect debug cable connection / disconnection in the non-overlapping regions.
 As an example use case of the two channel filters they can be used for detection of a USB-C debug accessory.
 The ADC must meet some minimum specifications:
-* Full scale range is 0.0V to 2.2V
-* If the signal is below 0.0V the ADC value will be zero.
-* If the signal is above 2.2V the ADC value will be maximum (i.e. same as 2.2V)
-* Absolute maximum error +/- 15 mV in the 0.25 - 0.45 V range
-* Absolute maximum error +/- 30 mV in the rest of the 0.0 - 2.2 V range
+* Full scale range must be 0.0V to 2.2V (note: the ADC in Earlgrey has a full range of 0.0V - 2.3V)
```

Comment:
How about: Full scale range must be 0.0V to <at least> 2.2V ...

Reply:
Sorry, I meant to say "... 0.0V to <AT LEAST> 2.2V ..."

Reply:
The list is about what is needed for the detection of the USB-C debug accessory. Where it is important to resolve down to 0.0V and up to 2.2V.

It should probably say "The full scale range must cover 0.0V to 2.2V" or "The full scale range must include 0.0V to 2.2V" or "The ADC must produce valid results in the range 0.0V to 2.2V". Then the parenthetical comment shows that the ADC in the Earlgrey part covers this range and a little more. (Of course if someone else was building another discrete chip from the Earlgrey confing and they had an ADC that connected to the controller and worked from 0.0V to 2.7V that would be fine too -- although they would need a different adjustment to the table)

Or split into two
* The minimum voltage for the ADC to produce a valid result is 0.0V or less
* The maximum voltage for the ADC to produce a valid result is 2.2V or more

Reply:
I tweaked the list as suggested in #21848 - PTAL